src/modules/taskUpdate.js

```diff
@@ -0,0 +1,19 @@
+import { tasksToDo, renderList, yetTodo } from './taskFunction.js';
+
+const checkedTask = ({ index, status }) => {
+  tasksToDo[index - 1].completed = status;
+  localStorage.setItem('tasksTodo', JSON.stringify(tasksToDo));
+  renderList();
+};
+
+const completedTasks = () => {
+  const uncompletedTodos = tasksToDo.filter((element) => element.completed !== true);
+  const newtodo = uncompletedTodos.map((element, index) => {
+    element.index = index + 1;
+    return element;
+  });
+  localStorage.setItem('tasksTodo', JSON.stringify(newtodo));
```

Comment:
- [ ] Good job so far 👍  But I have an issue, I click on any checkbox to change the completed status to true. I noticed there is another duplicate array that s created with the same info and then the completed status change but the first array remains the same. Please make sure that there is only one array in your local storage, you can check the screenshot below ⬇️  

![array](https://user-images.githubusercontent.com/6615394/229120871-7a5e2cb3-52a3-4544-92ff-79f596800b38.PNG)